quickannotator/api/v1/annotation/utils.py

```diff
@@ -32,6 +115,44 @@ def get_progress(self) -> float:
         with self.lock:  # Lock when accessing progress
             return (self.progress / self.total) * 100
 
+@ray.remote(max_concurrency=2)  # Add max_concurrency=2
+class AnnotationImporter(ProgressTracker): # Inherit from ProgressTracker
+    def __init__(self):
+        self.logger = LoggingManager.init_logger(constants.LoggerNames.RAY.value)
+        # step 1: import slide
+        # step 2: import annotations
+        super().__init__(len(2))  # Initialize ProgressTracker
```

Comment:
len(2) is invalid python:
```bash
jackson@jackson-Precision-7670:~$ python3 
Python 3.12.3 (main, Aug 14 2025, 17:47:21) [GCC 13.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> len(2)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: object of type 'int' has no len()

```

Reply:
Also, I think that hardcoding this value is error prone. Each increment should be directly tied to some kind of work, e.g, processing a column within the row.